src/main/resources/db/migration/V1_59__alter_judicial_location_mapping_table.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO dbjudicialdata.judicial_location_mapping (epimms_id,judicial_base_location_id,building_location_name,base_location_name,service_code) VALUES
```

Comment:
In ticket different schema is mentioned : dbjuddata , which one is correct ?

Reply:
There are now 2 schemas in Judicial DB. Need input from feature team, Magnus is on the case. This change is on hold for now.